### DISCUSSION TITLE
Error in compiling

### DISCUSSION THREAD

**Discussion by Quadrhyno** [Cyfrin/foundry-full-course-cu#1142] [Q&A]
am getting an error trying to deploy the foundry simple storage contract on my code editor. I used both commands provided in the course but i keep on getting error.
The Command 1: forge create SimpleStorage  --interactive
                         2: forge create SimpleStorage --rpc-url=http://127.0.0.1:8545 --private-key=0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d

The error: No files changed, compilation skipped
Error: 
error sending request for url (http://localhost:8545/): error trying to connect: tcp connect error: Connection refused (os error 111)

Context:
- Error #0: error trying to connect: tcp connect error: Connection refused (os error 111)
- Error #1: tcp connect error: Connection refused (os error 111)
- Error #2: Connection refused (os error 111)

**Comment by Niferu** [comment]
Hi @Quadrhyno 

You have to run your own blockchain local network first before deploying or running any scripts locally, so you need to do below:

1. Run command `anvil` ->this will host your local blockchain network
2. Then click + sign in terminal to open second window and there you should be able to run your scripts.

**Reply by Quadrhyno** [reply]
oh Have been able to resolve it. thank you so much

**Reply by Niferu** [reply]
@Quadrhyno 

Mark it as answer then, thanks :)
